assembl/static2/css/components/signup.scss

```diff
@@ -0,0 +1,27 @@
+.login-view {
+  .terms-link {
```

Comment:
is it different from other links ?

Reply:
Yes, the link had specific acceptance tests